Gopkg.lock

```diff
@@ -47,6 +42,11 @@
   packages = ["."]
   revision = "20b96f641a5ea98f2f8619ff4f3e061cff4833bd"
 
+[[projects]]
+  name = "github.com/intel/govmm"
+  packages = ["qemu"]
+  revision = "064ffdb2b27a49feea45eea8db1a6738f5dc4e09"
```

Comment:
This is buried in the revendor details, but I think it would help if you could you add a comment in the vendor commit that explains where `govmm` branched from `ciao` (basically steal the words from https://github.com/containers/virtcontainers/pull/510) as it's now also affecting the runtime.

Reply:
Yes will do !

Reply:
done